### PR TITLE
Update `jacoco-maven-plugin` to `0.8.8`

### PR DESCRIPTION
**Pull request by SerhatG**
This version has support for JDK17.

### CODE REVIEW

**Comment by SerhatG** [comment]
To hopefully fix: https://github.com/aerius/webdocument-exporter/actions/runs/10226171269

File: pom.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
     <aerius-tools.version>1.1.1</aerius-tools.version>
     <spotless.version>2.5.0</spotless.version>
-    <jacoco.version>0.8.6</jacoco.version>
+    <jacoco.version>0.8.8</jacoco.version>
```
Comment (BertScholten) [inline pom.xml]: might as well go to 0.8.12 right away? Higher java version support, and nothing breaking as far as I can tell.

**Review by BertScholten — APPROVED** [review]
ACK

File: pom.xml (hunk 1 above)
Comment (BertScholten) [inline pom.xml]:
```suggestion
    <jacoco.version>0.8.12</jacoco.version>
```

**Comment by SerhatG** [comment]
yeah I wanted to keep it in sync with `calculator` somehow.. not needed indeed

**Review by BertScholten — APPROVED** [review]
ACK

**Pull request merged**